go/vt/vtgate/planbuilder/testdata/unsupported_cases.json

```diff
@@ -37,12 +37,12 @@
   {
     "comment": "join with USING construct",
     "query": "select * from user join user_extra using(id)",
-    "plan": "can't handle JOIN USING without authoritative tables"
+    "plan": "VT03019: column id not found"
```

Comment:
this doesn't seem correct. we don't know that the column doesn't exist, we just don't have enough info to assert it, right? shouldn't we keep the old error message here?

Reply:
solved via a07c5dfbc0